toolbox/release_note_collector/main.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-github/github"
+	u "istio.io/test-infra/toolbox/util"
+)
+
+const (
+	releaseNoteSuffix = ".releasenote"
+)
+
+var (
+	org       = flag.String("user", "istio", "Github owner or org")
+	repos     = flag.String("repos", "", "Github repos, separate using \",\"")
+	label     = flag.String("label", "release-note", "Release-note label")
+	sort      = flag.String("sort", "create", "The sort field. Can be comments, created, or updated.")
+	order     = flag.String("order", "desc", "The sort order if sort parameter is provided. One of asc or desc.")
+	version   = flag.String("version", "", "Release version")
+	output    = flag.String("output", "./", "Path to output file")
+	startDate = flag.String("start_date", "", "Start date")
+	endDate   = flag.String("end_date", "", "End date")
+)
+
+func main() {
+	flag.Parse()
+
+	repoList := strings.Split(*repos, ",")
+	for _, repo := range repoList {
+		log.Printf("Start fetching release note from %s", repo)
+		queries := createQueryString(repo)
+		log.Printf("Query: %s", queries)
```

Comment:
`queries` is a slice and I was not sure what is the behavior of %s in this case. Maybe %v?